src/admin-views/tribe-options-display.php

```diff
@@ -256,6 +256,7 @@ static function ( $view ) {
 $sample_date = strtotime( 'January 15 ' . gmdate( 'Y' ) );
 
 $end_time_options = [
+	'recent'       => esc_html_x( 'Recent past events list', 'The option to remove end times for the past events page view.', 'the-events-calendar' ),
```

Comment:
Why do we use 'recent' here? The slug for LP should be [latest-past](https://github.com/the-events-calendar/the-events-calendar/blob/master/src/Tribe/Views/V2/Views/Latest_Past_View.php#L26)

Reply:
Of course, then we'll have to change this as well:
https://github.com/the-events-calendar/the-events-calendar/blob/master/src/Events/Views/Modifiers/Hide_End_Time_Modifier.php#L66

Reply:
It's historical, we had this as an extension and it called it `recent`. I'll update it.